.tags

```diff
@@ -0,0 +1,31 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
This file shouldn't be committed - you can remove it with `git rm .tags` and then add `.tags` to the `.gitignore` file and commit that.